backend/app/services/llm/jobs.py

```
@@ -13,8 +13,22 @@
+    LLMCallConfig,
+    KaapiCompletionConfig,
+    TextContent,
+    TextInput,
+    Validator,
+)
+from app.services.llm.guardrails import (
+    list_validators_config,
+    run_guardrails_validation,
+)
+from app.services.llm.guardrails import (
+    list_validators_config,
```

Ruff (0.15.0)

[error] Redefinition of unused `list_validators_config` from line 25: `list_validators_config` redefined here

Remove definition: `list_validators_config`

(F811)